renovate-presets/drupal-10.json5

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Let's not have a version specific one since we're doing the bare minimum configuration for Drupal best practices that we'll push up to Renovate defaults.